### DISCUSSION TITLE
Redash Scheduler not working

### DISCUSSION THREAD

**Discussion by amrendraupadhyay** [getredash/redash#6658] [Q&A]
I am using Redash 11 version. In this version i am unable to schedule queries. Can anyone suggest why i am facing such issue and how to fix it?
Although in redash 8 version schedule queries is running properly on time but SSO login support is not there.
I want SSO login, Oracle support and Schedule queries to run on time. Which version should i use?

**Comment by masayuki038** [comment]
Hi @amrendraupadhyay,

We have discussed the scheduler issue two weeks ago. Please check https://github.com/getredash/redash/issues/6369#issuecomment-1826882043.
